components/common/ProfileImage.tsx

```diff
@@ -3,13 +3,23 @@ import Image from 'next/image';
 export interface ProfileImageProps {
   name: string;
   src: string | null;
+  width?: number;
+  height?: number;
 }
 
-export default function ProfileImage({ name, src }: ProfileImageProps) {
+export default function ProfileImage({
+  name,
+  src,
+  width = 38,
+  height = 38,
+}: ProfileImageProps) {
   return (
-    <div className="rounded-full flex-center w-38pxr h-38pxr bg-green">
+    <div
+      className={`rounded-full flex-center bg-green`}
+      style={{ width: width, height: height }}
```

Comment:
방법1
```suggestion
profileImage ({name, src, size}) //가로 세로 동일하므로 속성값으로 size만 받음 {
const size = "w-{size} h-{size}"
      className={`rounded-full flex-center bg-green ${size}`}
```
style로 빼야하는 것이 아쉽다면 위와 같은 방법도 있을 것 같아요!!

Reply:
ㅜㅜ 이거 검색해봤는데..
"Tailwind CSS에서 동적인 값을 클래스 이름으로 직접 삽입하는 것은 지원하지 않습니다. Tailwind는 빌드 타임에 모든 클래스를 생성하므로, 런타임에 클래스 이름을 만들어내는 것은 작동하지 않습니다. " 라네요.
JIT모드를 설정하라는데, 가능하면 추후 리팩토링 하겠습니다 ㅠ

Reply:
헉 그렇군요!! 저희 colorChip 등에서는 위랑 비슷한 방식을 사용해서 가능할 줄 알았어요🥲
찾아봐주셔서 감사합니다 ㅠㅠ

Reply:
로직으로 처리를 해놓은다음에 그걸 className에 할당하는 방식은 될것같아서 그렇게 수정해봤어요.
size에 'sm'입력하면 작은 사이즈(26px)로, 별도로 사이즈를 지정하지않으면 기본 크기(38px)로 표시되도록 수정했습니댜